jalali_date/settings.py

```diff
@@ -3,6 +3,7 @@
 from django.conf import settings
 
 JALALI_DATE_DEFAULTS = {
+    'LIST_DISPLAY_AUTO_CONVERT': False,
```

Comment:
we can talk together about the side effects of this feature and the future of the library.

please message me (Telegram: armanroomana, Gmail: roomana.arman@gmail.com)